packages/app/src/cli/services/context.ts

```diff
@@ -585,9 +598,65 @@ async function logMetadataForLoadedDeployContext(env: DeployContextOutput) {
   }))
 }
 
+export async function enableDeveloperPreview(app: OrganizationApp, token: string) {
+  return developerPreviewUpdate(app, token, true)
+}
+
+export async function disableDeveloperPreview(app: OrganizationApp, token: string) {
+  return developerPreviewUpdate(app, token, false)
+}
+
+async function developerPreviewUpdate(app: OrganizationApp, token: string, enabled: boolean) {
+  if (!app.betas?.unifiedAppDeployment) return
+
+  const tasks = [
+    {
+      title: `${enabled ? 'Enabling' : 'Disabling'} developer preview...`,
+      task: async () => {
+        let result: DevelopmentStorePreviewUpdateSchema | undefined
+        let error: string | undefined
+        try {
+          const query = DevelopmentStorePreviewUpdateQuery
+          const variables: DevelopmentStorePreviewUpdateInput = {
+            input: {
+              apiKey: app.apiKey,
+              enabled,
+            },
+          }
+          result = await partnersRequest(query, token, variables)
+          // eslint-disable-next-line no-catch-all/no-catch-all, @typescript-eslint/no-explicit-any
+        } catch (err: any) {
+          error = err.message
+        }
+
+        if ((result && result.developmentStorePreviewUpdate.userErrors?.length > 0) || error) {
+          const previewURL = outputToken.link(
+            'partners dashboard',
+            await devPreviewURL({orgId: app.organizationId, appId: app.id}),
+          )
+          outputWarnError(
+            outputContent`Unable to ${
+              enabled ? 'enable' : 'disable'
+            } developer store preview for this app. You can change this setting in the ${previewURL}.'}`,
```

Comment:
Important to be precise in the language of this setting :D
```suggestion
            } development store preview for this app. You can change this setting in the ${previewURL}.'}`,
```